internal/cmd/exodus.go

```diff
@@ -16,11 +16,25 @@ import (
 	"github.com/release-engineering/exodus-rsync/internal/walk"
 )
 
-func webURI(srcPath string, srcTree string, destTree string) string {
+func webURI(srcPath string, srcTree string, destTree string, strip string) string {
 	cleanSrcPath := path.Clean(srcPath)
 	cleanSrcTree := path.Clean(srcTree)
 	relPath := strings.TrimPrefix(cleanSrcPath, cleanSrcTree+"/")
 
+	// If the configured strip string contains ":", any characters following the ":"
+	// must be stripped from the destination path.
+	// For example, if exodus-rsync is invoked with
+	//       "exodus-rsync ./src/ otherhost:/foo/bar/baz/my/dest",
```

Comment:
```suggestion
	// "exodus-rsync ./src/ otherhost:/foo/bar/baz/my/dest",
```
Or was the spacing intentional?

Reply:
It was intentional but I also thought it looked a bit strange, I'll remove it